.palantir/revapi.yml

```diff
@@ -6099,6 +6099,11 @@ acceptedBreaks:
       new: "method void misk.redis.Redis::pipelining(kotlin.jvm.functions.Function1<?\
         \ super misk.redis.DeferredRedis, kotlin.Unit>)"
       justification: "Adds proper support for pipelining Redis commands"
+  "2024.03.21.182031-f6f5775":
+    com.squareup.misk:misk-redis:
+    - code: "java.method.removed"
+      old: "method redis.clients.jedis.Pipeline misk.redis.Redis::pipelined()"
```

Comment:
I found no usages of this method in our internal repositories.

Now that we have `pipelining {}` to replace `pipelined()`, we should probably think about cutting some releases.

Perhaps cut one before this change, so folks can pick up the deprecation, and then another one afterwards?